### PR TITLE
some random mappings

### PR DESCRIPTION
**Pull request by johannes-schmatz**

### CODE REVIEW

File: mappings/1.11.2#17w06a.tinydiff [hunk 1]
```diff
@@ -39,7 +39,7 @@ c	net/minecraft/unmapped/C_5904422
 	m	(Lnet/minecraft/unmapped/C_9590849;)Z	m_9015927		m_9015927
 		p	1			p_1
 c	net/minecraft/unmapped/C_6260518
-	m	(Lnet/minecraft/unmapped/C_8105098;I)V	m_4993396		m_4993396
+	m	(Lnet/minecraft/unmapped/C_8105098;I)V	m_4993396		saveOrLoadToolbar
```
Comment (SpaceWalkerRS) [inline mappings/1.11.2#17w06a.tinydiff:42]: I'm not able to look at the code right now so this name is a little confusing to me. Is it like a toggle?

Comment (johannes-schmatz) [reply mappings/1.11.2#17w06a.tinydiff:42]: code is basically ```java
boolean save = saveKey.isPressed();
boolean load = loadKey.isPressed();
saveOrLoadToolBar(..., save, load)```
(at least in 1.12.2)

Comment (SpaceWalkerRS) [reply mappings/1.11.2#17w06a.tinydiff:42]: I see, so it's a bit of a mojank moment then.

**Review by SpaceWalkerRS — APPROVED** [review]

**Comment by SpaceWalkerRS** [comment]
looks good, I'll wait for the workflow to run and merge it

**Pull request merged**